### PR TITLE
net/netmgr: add socket trace log

### PR DESCRIPTION
**Pull request by pillip8282**
add socket trace log

### CODE REVIEW

File: os/net/netmgr/netstack_lwip.c [hunk 1]
```diff
@@ -85,6 +95,40 @@ static int _socket_argument_validation(int domain, int type, int protocol)
 	return 0;
 }
 
+static void _get_port(int fd, int *port)
+{
+	*port = -1;
+	struct lwip_sock *sock = get_socket(fd, getpid());
+	if (!sock) {
+		return;
```
Comment (jeongchanKim) [inline os/net/netmgr/netstack_lwip.c:103]: How about adding logs for failure?

File: os/net/netmgr/netstack_lwip.c [hunk 2]
```diff
@@ -85,6 +95,40 @@ static int _socket_argument_validation(int domain, int type, int protocol)
...
+		*port = upcb->local_port;
+	}
+}
+
+static void _get_proto(int fd, char **type)
+{
+	*type = UNKNOWN_STR;
+	struct lwip_sock *sock = get_socket(fd, getpid());
+	if (!sock) {
+		return;
```
Comment (jeongchanKim) [inline os/net/netmgr/netstack_lwip.c:121]: Same as above.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**